R/codecov.R

```diff
@@ -0,0 +1,143 @@
+#' Run covr on a package and upload the result to codecov.io
+#' @param path file path to the package
+#' @param Codecov url (change for Enterprise)
+#' @param ... additional arguments passed to \code{\link{package_coverage}}
+#' @export
+codecov <- function(path = ".", base_url = "https://codecov.io", ...) {
+  
+  # -------
+  # Jenkins
+  # -------
+  if (Sys.getenv("JENKINS_URL")) {
+    # https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project
+    # path <- Sys.getenv("WORKSPACE")
+    codecov_url <- paste0(base_url, "/upload/v2?service=jenkins",
+                          "&branch=", Sys.getenv("GIT_BRANCH"),
+                          "&commit=", Sys.getenv("GIT_COMMIT"),
+                          "&build=", Sys.getenv("BUILD_NUMBER"),
+                          "&build_url=", Sys.getenv("BUILD_URL"))
+  # ---------
+  # Travis CI
+  # ---------
+  } else if (Sys.getenv("CI") == "true" && Sys.getenv("TRAVIS") == "true") {
```

Comment:
Not sure if it has to be in an `else` branch -- IMHO it should work fine (and indentation would be easier) with closing the previous `if` conditional and starting a new one